### PR TITLE
Minor fixes with Figure 2 quality and texts 

### PR DESCRIPTION
**Pull request by axiomcura**
This pull request addresses the reviewers' minor suggestions:

>"1. It would be beneficial to improve the quality of figures to high resolution, especially for Figure. 2B, 2C, and 2D.
>2. The numbers on the top row of Figure. 2C and Supplementary Figure 3 are too crowded and difficult to read.
>3. Consistent decimal points should be used for labeling the training and testing performances for Figure 2B. "

These changes are implemented within the `4.visualization` module, with the goal of improving both figure quality and readability as recommended by the reviewers.

### CODE REVIEW

File: notebooks/4.visualization/nbconverted/4.visualization.r
Comment (gwaybio) [inline notebooks/4.visualization/nbconverted/4.visualization.r]: to delete

File: output.png
Comment (gwaybio) [inline output.png:1]: The font is still a bit too small. Here's a few suggestions:

## Panel B

- Move the figure legend for panel B to the lower right facet (below proteasome)
- Increase the font size of text within each box. You can remove the "F1 Score" label above and simply use: "F1 train: XX\nF1 test: YY"
- Increase the font size of the facet labels. You can rename "Nonspecific reactive" to "Nonspecific" if you need to save space
- Increase the font for axis titles and labels.
- Slightly increase the width of the lines

## Panel C

- The number within each box is still very difficult to read. Moving the legend in panel B will help.
- Renaming "Nonspecific reactive" to "Nonspecific" will also help
- You can also move the "Ratio" legend to underneath the plot

## Panel D

- Increase height of the plot
- Add more line breaks to labels on the y axis: "Cytoskeletal injury\ncompound\nground truth" (for example)
- Increase size of axis titles and labels

**Review by gwaybio — APPROVED** [review]
This is a really important comment by the reviewer, and we need to present a figure that removes all doubt. There are still some tweaks that we need to make. See my responses below. We should chat as well if anything is unclear

**Comment by axiomcura** [comment]
@gwaybio I’ve applied all requested changes to the main Figure 2. Once I get your approval, I’ll proceed with updating all figures in the manuscript!

**Review by gwaybio — APPROVED** [review]

**Comment by axiomcura** [comment]
Merging!

**Comment by gwaybio** [comment]
finalized figure in #35

**Pull request merged**
